ddprofiler/app/src/main/java/ddprofiler/core/Profile.java

```diff
@@ -1,23 +1,52 @@
 package ddprofiler.core;
 
-public record Profile (long id,
-                       String dbName,
-                       String path,
-                       String sourceName,
-                       String columnName,
-                       String dataType,
-                       String columnLabel,
-                       int totalValues,
-                       int uniqueValues,
-                       int nonEmptyValues,
-                       String entities,
-                       long[] minhash,
-                       String xstructure,
-                       float minValue,
-                       float maxValue,
-                       float avgValue,
-                       long median,
-                       long iqr) {
+public record Profile(
+        long id,
+        String dbName,
+        String path,
+        String sourceName,
+        String columnName,
+        String dataType,
+        String columnLabel,
+        int totalValues,
+        int uniqueValues,
+        int nonEmptyValues,
+        String entities,
+        long[] minhash,
+        String xstructure,
+        float minValue,
+        float maxValue,
+        float avgValue,
+        long median,
+        long iqr,
+        String semanticType
```

Comment:
If I were to interpret this function signature, I would assume that semanticType is a string without any additional fields.

Reply:
Changed at 44c3adbcb6fb3d07c972c2708c40dcd5207d91c9